### PR TITLE
[VMware] Fix service offerings listing on appliances deployment options

### PR DESCRIPTION
**Pull request by nvazquez**
### Description

The service offerings matching the deployment options requirements were not listed, except unconstrained offerings. This PR fixes the listing of service offerings on VMware deployment options:

Screen Shot 2021-12-06 at 12 00 07

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by sureshanaparti** [comment]
@blueorangutan package

**Comment by shwstppr** [comment]
@nvazquez can you please elaborate what behaviour this PR fixes

**Comment by sureshanaparti** [comment]
@blueorangutan package

**Comment by nvazquez** [comment]
@blueorangutan package

**Review by rhtyd — APPROVED** [review]

**Comment by nvazquez** [comment]
@blueorangutan test centos7 vmware-67u3

**Review by sureshanaparti — APPROVED** [review]
code LGTM

**Comment by sureshanaparti** [comment]
@blueorangutan test centos7 vmware-67u3

File: ui/src/views/compute/wizard/ComputeOfferingSelection.vue [hunk 1]
```diff
@@ -157,11 +157,11 @@ export default {
...
           disabled = true
         }
-        if (disabled === false && this.minimumCpuspeed > 0 && maxCpuSpeed && maxCpuSpeed !== this.minimumCpuspeed) {
+        if (disabled === false && this.minimumCpuspeed > 0 && maxCpuSpeed && maxCpuSpeed < this.minimumCpuspeed) {
           disabled = true
         }
         if (disabled === false && maxMemory && this.minimumMemory > 0 &&
-          ((item.iscustomized === false && maxMemory !== this.minimumMemory) ||
+          ((item.iscustomized === false && maxMemory < this.minimumMemory) ||
             (item.iscustomized === true && maxMemory < this.minimumMemory))) {
```
Comment (DaanHoogland) [inline ui/src/views/compute/wizard/ComputeOfferingSelection.vue:165]: so,
```suggestion
          (maxMemory < this.minimumMemory) //  && (item.iscustomized === false || item.iscustomised == true)
```
?

Comment (sureshanaparti) [reply ui/src/views/compute/wizard/ComputeOfferingSelection.vue:165]: > so,
> 
> ?

+1, `maxMemory < this.minimumMemory` should be fine. @nvazquez can you check and update.

**Review by rohityadavcloud — APPROVED** [review]

**Pull request merged**